config/manager/kustomization.yaml

```diff
@@ -1,5 +1,4 @@
 resources:
-- webhook_secret.yaml
```

Comment:
@gaohoward This is likely a result of the ENABLE_WEBHOOKS work as it gets dynamically added and removed based on the value assigned to ENABLE_WEBHOOKS. Probably doesn't matter if it's included or not; with the default being false maybe it's good to have it not set for the moment.

Reply:
yeah I think that's the result of the ENABLE_WEBHOOK flag. I'll restore it.